packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details/paths.dart

```diff
@@ -114,18 +115,19 @@ class _RetainingPathTableColumns {
 }
 
 class RetainingPathTable extends StatelessWidget {
-  const RetainingPathTable({
+  RetainingPathTable({
     Key? key,
-    required this.entries,
+    required this.classData,
     required this.selection,
     required this.isDiff,
-    required this.className,
   }) : super(key: key);
 
-  final List<StatsByPathEntry> entries;
-  final ValueNotifier<StatsByPathEntry?> selection;
+  final ValueNotifier<PathData?> selection;
   final bool isDiff;
-  final String className;
+  final ClassData classData;
+
+  late final _data =
```

Comment:
Number of paths is not huge. Average is ~5, 95th percentile is like 30.
What alternative to you suggest? We do not want to calculate it upfront for each class for each path, because it will take memory. So, it will happen on user click anyway. That is build cycle. Or we will need to make it async, with spinning wheal, but i want to see confirmation of the issue before introducing complexity.

Reply:
If it is fast, then this is okay to have in build. But we should get some numbers to make sure.

Reply:
Can you suggest numbers that will convince you?

Reply:
Kenzie: We only have a 16 ms budget for the UI portion of a flutter frame in order to maintain 60 FPS. And this build method is not the only one getting triggered for a single frame, so if it takes more than 2 ms I think it would be worth digging a bit further to see if we are causing UI jank. And you'll want to check this on Chrome in profile mode, not desktop.

Reply:
Tested for it in devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details/paths.dart